static/bidder-info/inmobi.yaml

```diff
@@ -6,6 +6,7 @@ capabilities:
     mediaTypes:
       - banner
       - video
+      - native
```

Comment:
Just to confirm, you want the native support only for `app` and not for `site`, right?

Reply:
Correct.